.github/workflows/ci.yml

```diff
@@ -7,19 +7,31 @@ jobs:
     runs-on: ubuntu-latest
     name: Ubuntu Bionic CI
     steps:
+      - name: Login to GitHub Container Registry
```

Comment:
Is there a way we can skip this step? Can the container be publicly available, or shared with the `ignitionrobotics` org? This way, we just need to update the `action-ignition-ci` repository and don't need to add this step to all branches of all libraries.